src/pages/get-empowered.js

```diff
@@ -2,32 +2,44 @@ import React from "react"
 import SEO from "../components/seo"
 import Layout from "../components/layout"
 import {Link, graphql} from "gatsby"
-// import TertiaryNav from "../components/TertiaryNav.jsx"
+import TertiaryNav from "../components/TertiaryNav.jsx"
 
 
 const EmpowerPage = ({ data }) => {
     const micrositeData = data.allMicrositesJson.edges.map(n => n.node)
-    
+  
+  const toSlug = (string) => {
+    return string.replace(/\s/g, '-').toLowerCase()
+  }
+
   return (
     <Layout>
       <SEO title="Get Empowered" />  
-      <div className="side-wrapper">
-        <div className="container main">
-          <div className="content">
-          <h1>Get Empowered</h1>
-            {
-              micrositeData.map(({title, description, fields, content}, idx) => {
-                  return (
-                    <div className="microsite-paragraph" key={idx}>
-                        <h3 name={fields.slug.slice(1)}>{title}</h3>
-                        <p>{description}</p>
-                        <Link to={`/get-empowered/${fields.slug}/${content[0].title.replace(/\s/g, '-').toLowerCase()}`}>View Site</Link>
-                    </div>
-                  )
-              }) 
-            }
+      <div className="tertiary-background">
+        <div className="side-wrapper">
+          <div className="container main">
+            <div className="content">
+            <h1>Get Empowered</h1>
+              {
+                micrositeData.map(({title, description, fields, content}, idx) => {
```

Comment:
Don't need `return` keyword or curly brackets

```suggestion
                micrositeData.map(({title, description, fields, content}, idx) => (<div className="microsite-paragraph" key={idx}>
                ...
```